app/src/main/res/values/strings.xml

```diff
@@ -74,4 +74,7 @@
     <string name="title_add">Add</string>
     <string name="title_create">Create</string>
     <string name="action_x">x</string>
+    <string name="rp_bt_sales">Sales</string>
```

Comment:
3.2.2 Strings

String names start with a prefix. For example registration_email_hint or registration_name_hint. If a string doesn't belong to any section, follow rules here:
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md